cdk/domino_cdk/config/iam.py

```diff
@@ -351,11 +366,6 @@ def do_cf():
                 "elasticfilesystem:DescribeAccessPoints",
                 "elasticfilesystem:DescribeFileSystems",
                 "iam:ListAttachedRolePolicies",
-                "kms:CreateGrant",
-                "kms:Decrypt",
-                "kms:DescribeKey",
-                "kms:GenerateDataKey",
-                "kms:RetireGrant",
```

Comment:
These deletions remove it from the cloudformation version of the iam policies.

Reply:
He adds the whole "kms:* on * if from CF" block to base

Reply:
> from the cloudformation version 

you mean "cdk deploy" version

Reply:
Yes, cdk deploy is what I meant by that.

Reply:
We need some terminology.

I know:
* CloudFormation UI = manual
* Terraform = manual
* cdk deploy = not manual

When you say "the cloudformation version" I think CF UI

Reply:
Manual/not manual is what I went with in the code to avoid that ambiguity, but in my head I think of it as cloudformation/terraform and that leaked out.